src/error.rs

```diff
@@ -122,18 +122,84 @@ impl fmt::Display for ErrorType {
 #[derive(Debug, PartialEq, Deserialize)]
 #[serde(rename_all = "snake_case")]
 pub enum ErrorCode {
-    InvalidNumber,
-    InvalidExpiryMonth,
-    InvalidExpiryYear,
-    InvalidCvc,
-    InvalidSwipeData,
-    IncorrectNumber,
+    AccountAlreadyExists,
+    AccountCountryInvalidAddress,
+    AccountInvalid,
+    StripeAccount,
+    AccountNumberInvalid,
+    AlipayUpgradeRequired,
+    AmountTooLarge,
+    AmountTooSmall,
+    ApiKeyExpired,
+    BalanceInsufficient,
+    BankAccountExists,
+    BankAccountUnusable,
+    BankAccountUnverified,
+    BitcoinUpgradeRequired,
+    CardDeclined,
+    DeclineCode,
+    ChargeAlreadyCaptured,
+    ChargeAlreadyRefunded,
+    ChargeDisputed,
+    ChargeExpiredForCapture,
+    CountryUnsupported,
+    CouponExpired,
+    CustomerMaxSubscriptions,
+    EmailInvalid,
     ExpiredCard,
+    IncorrectAddress,
     IncorrectCvc,
+    IncorrectNumber,
     IncorrectZip,
-    CardDeclined,
-    Missing,
+    InstantPayoutsUnsupported,
+    InvalidCardType,
+    InvalidChargeAmount,
+    InvalidCvc,
+    InvalidExpiryMonth,
+    InvalidExpiryYear,
+    InvalidNumber,
+    InvalidSourceUsage,
+    InvoiceNoCustomerLineItems,
+    InvoiceNoSubscriptionLineItems,
+    InvoiceNotEditable,
+    InvoiceUpcomingNone,
+    LivemodeMismatch,
+    OrderCreationFailed,
+    OrderRequiredSettings,
+    OrderStatusInvalid,
+    OrderUpstreamTimeout,
+    OutOfInventory,
+    ParameterInvalidEmpty,
+    ParameterInvalidInteger,
+    ParameterInvalidStringBlank,
+    ParameterInvalidStringEmpty,
+    ParameterMissing,
+    ParameterUnknown,
+    PaymentMethodUnactivated,
+    PayoutsNotAllowed,
+    PlatformApiKeyExpired,
+    PostalCodeInvalid,
     ProcessingError,
+    ProductInactive,
+    RateLimit,
+    ResourceAlreadyExists,
+    ResourceMissing,
+    RoutingNumberInvalid,
+    SecretKeyRequired,
+    SepaUnsupportedAccount,
+    ShippingCalculationFailed,
+    SkuInactive,
+    StateUnsupported,
+    LegalEntity,
+    TaxIdInvalid,
+    TaxesCalculationFailed,
+    TestmodeChargesOnly,
+    TlsVersionUnsupported,
+    TokenAlreadyUsed,
+    TokenInUse,
+    TransfersNotAllowed,
+    UpstreamOrderCreationFailed,
+    UrlInvalid
 }
```

Comment:
Probably worth using the `#[doc(hidden)] __NonExhaustive` pattern here since more error codes are likely in the future